.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,19 @@
+name: Continous Integration
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        version: 10.x
```

Comment:
Why does it have to be 10.x?

Asking to learn; not a PR blocker

Reply:
Don't think it has to be, just better to be specific, otherwise I think you just get latest LTS every time

Reply:
this actually does have to be 10.x, it uses a native module that won't compile on 12